import.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-RUST_BACKTRACE=1 cargo run --release --manifest-path importer/Cargo.toml --features scenarios -- $@;
+RUST_BACKTRACE=1 cargo run --bin importer --release --manifest-path importer/Cargo.toml --features scenarios -- $@;
```

Comment:
Ah, I thought there was one `main` per crate. I like this, will probably merge `iotool` into `importer` as well.